src/core.js

```diff
@@ -403,11 +390,17 @@ define([
             });
         });
 
+        // Section toggling events: within section changer and when clicking on section header
```

Comment:
I think this means there are now two ways to expand/collapse a section group: clicking the section header and also new items in the question properties menu. Is that right? Out of curiosity, how was it determined that the items in the question properties menu were needed?